metadata-db-app/src/cmr/metadata_db/data/oracle/service_table.clj

```diff
@@ -1,118 +0,0 @@
-(ns cmr.metadata-db.data.oracle.service-table
```

Comment:
This ns wasn't used anywhere. so I removed it.

Reply:
Good call. This was probably here before when we had the idea of a separate table for services for each provider and so on provider creation we needed to create the table.